libs/response/src/lib/typings/odds/odds-response.ts

```diff
@@ -33,6 +35,16 @@ export interface OddsResponse {
    */
   bestOffers?: OddsBestOffer[];
 
+  /**
+   *  Limited list of supported team sports
+   */
+  sport: Sports;
+
+  /**
+   * Ordering of odds in the odds list
+   */
+  oddsOfferOrder: OddsOfferType[];
```

Comment:
Please make it optional and provide a default order in the server if it was not provided from server. Just in case if new version if SDK talks with old version of API.

Reply:
Is this really a use-case? As we are hosting the API I see no real value in having the SDK being API backwards compatible. For me this would be just technical dept.

Reply:
It really can be the case. Release cycles are a bit different for different environments (we already have three: UR Dev, US Prod and in Australia). So after release of this version of SDK it might be that corresponding baclend functionality is not released yet.

Reply:
Ok, done